debian/entry.sh

```diff
@@ -83,6 +83,7 @@ function init_non_systemd()
 		fi
 		"$CMD" "$@" &
 		pid=$!
+		wait "$pid"
```

Comment:
@petrosagg yeah, `fg` should be removed. The explanation was added in the original commit https://github.com/resin-io-library/resin-rpi-raspbian/pull/48, I also added it here